content/en/docs/setup/upgrade/in-place/index.md

```diff
@@ -17,6 +17,11 @@ The upgrade command can also perform a downgrade of Istio.
 See the [`istioctl` upgrade reference](/docs/reference/commands/istioctl/#istioctl-upgrade)
 for all the options provided by the `istioctl upgrade` command.
 
+{{< warning >}}
+`istioctl upgrade` is for in-place upgrade and not compatible with `revisions`. If you installed the previous version
+of Istio with the `--revision` flag and try to upgrade the next version with `istioctl upgrade`, it will fail.
```

Comment:
"will fail" sounds pretty bad. What exactly does it do? Does it corrupt things, or does it generate an error message that says it can't be done?

cc @ostromart

Reply:
More details in PR description https://github.com/istio/istio/issues/28566

Reply:
We are working on it to improve error message.

Reply:
"not compatible with installations done with the --revision flag. Upgrades of such installation will fail with an error."